usagov/vote-gov-drupal
# Vote-1124: Responsive Module Set Up

## Jira ticket

[Vote-1124](https://cm-jira.usa.gov/browse/VOTE-1124)

## Description

This Pr adds and configures the responsive image module from drupal.  This will address a lighthouse error that was flagged and will help set up better performance for future images as well  

## Deployment and testing

### Post-deploy steps

1. run `lando retune` and cd into the `votegov` theme and run `npm run build`

### QA/Testing instructions

1. log in as an admin and add an image to the hero of the home page... verify that there are no major visual regressions on various breakpoints 
2. open up console and run google light house on mobile and verify that the error `Properly size images` is no longer getting flagged 

## Checklist for the Developer




## Checklist for the Peer Reviewers
